lib/spack/docs/index.rst

```diff
@@ -65,6 +65,7 @@ or refer to the full manual below.
    repositories
    command_index
    package_list
+   gpg
```

Comment:
Ah, I see. Maybe Basic Usage then once it's up and working? I imagine most users will want to be able to download binary packages from LLNL.